lib/mintacoin/accounts/crypto/stellar.ex

```diff
@@ -49,14 +49,12 @@ defmodule Mintacoin.Accounts.Stellar do
   @spec build_create_account_operation(destination :: public_key()) ::
           create_account()
   defp build_create_account_operation(destination) do
-    starting_balance =
-      @starting_balance
-      |> String.to_float()
-
-    CreateAccount.new(
-      destination: destination,
-      starting_balance: starting_balance
-    )
+    @starting_balance
+    |> String.to_float()
+    |> (&CreateAccount.new(
+          destination: destination,
+          starting_balance: &1
+        )).()
```

Comment:
@Odraxs let's update the code as you proposed before, I think looks better
```suggestion
starting_balance =  String.to_float(@starting_balance)

CreateAccount.new(
  destination: destination,
  starting_balance: starting_balance
)
```
I apologize for this change.